charts/back8sup/values.yaml

```diff
@@ -0,0 +1,92 @@
+# Default values for back8sup.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: adfinissygroup/back8sup
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart version.
```

Comment:
Looking at https://github.com/adfinis-sygroup/helm-charts/pull/23/files#diff-de811fa87fb1a013304fca09546338adR46 I think this comment is incorrect.

Reply:
It looks like the chart was initialized with an early version of Helm 3 tbh.

Reply:
```
version.BuildInfo{Version:"v3.2.0", GitCommit:"e11b7ce3b12db2941e90399e874513fbd24bcb71", GitTreeState:"clean", GoVersion:"go1.14.2"}
```

Reply:
Looks like yours is newer than mine and the defaults have changed yet again :P

Reply:
Helm 3 usually generates this version:
```yaml
          image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
```

This brings up the question of the need for some tags and release management in the back8sup repo.